docs/getting-started/cli-getting-started.md

```diff
@@ -119,12 +134,14 @@ For more information, see [Writing scripts](../user-guide/cli-using-writing-scri
 
 ## Next steps
 
-You successfully installed Zowe CLI, issued your first commands, and wrote a simple script! Next, you might want to perform the following tasks:
+You successfully installed Zowe CLI, issued your first commands, and wrote a simple script! 
+
+Next, you might want to perform the following tasks:
 
 - Issue the `zowe --help` command to explore the product functionality, or review the online [web help](../user-guide/cli-using-displaying-help.md).
-- Learn how to configure Zowe CLI [run Zowe CLI in daemon mode](../user-guide/cli-using-using-daemon-mode.md). **Daemon mode** significantly improves the performance of Zowe CLI commands by running Zowe CLI as a persistent background process.
+- Learn how to configure Zowe CLI [to run in daemon mode](../user-guide/cli-using-using-daemon-mode.md). **Daemon mode** significantly improves the performance of Zowe CLI commands by running Zowe CLI as a persistent background process.
 - Learn about [configuring environment variables](../user-guide/cli-configuringcli-ev.md) to store configuration options.
-- Learn about [integrating with API Mediation Layer](../user-guide/cli-using-integrating-apiml.md).
+- Learn about [using API Mediation Layer](../user-guide/cli-using-integrating-apiml.md).
```

Comment:
is this one of the instances where we should use `the` API ML...?

Reply:
No